frontend/src/container/OnboardingContainer/APM/Java/md-docs/jboss.md

```diff
@@ -35,9 +35,9 @@ Step 3. Update `JAVA_OPTS` environment variable
 Update `JAVA_OPTS` environment variable with configurations required to send data to SigNoz cloud in your configuration file.
 
 ```bash
-JAVA_OPTS="-javaagent:/<path>/opentelemetry-javaagent.jar
--Dotel.exporter.otlp.endpoint=https://ingest.{region}.signoz.cloud:443
--Dotel.exporter.otlp.headers="signoz-access-token=<SIGNOZ_INGESTION_KEY>"
+JAVA_OPTS="-javaagent:/<path>/opentelemetry-javaagent.jar \
+-Dotel.exporter.otlp.endpoint=https://ingest.{region}.signoz.cloud:443 \
+-Dotel.exporter.otlp.headers="signoz-access-token=<SIGNOZ_INGESTION_KEY>" \
```

Comment:
Did you run this and verify? I am assuming this is not needed for `JAVA_OPTS`.

Reply:
> Did you run this and verify? I am assuming this is not needed for `JAVA_OPTS`.

No, I didn't run and verify this.

We would be exporting this command as environment variable in the `standalone.conf` file 

So either we can do 
```bash
export JAVA_OPTS="$JAVA_OPTS -javaagent:/<path>/opentelemetry-javaagent.jar -Dotel.exporter.otlp.endpoint=https://ingest.{region}.signoz.cloud:443 -Dotel.exporter.otlp.headers='signoz-access-token=<SIGNOZ_INGESTION_KEY>' -Dotel.resource.attributes='service.name=<app_name>'"
```
OR

```bash
export JAVA_OPTS="$JAVA_OPTS \
-javaagent:/<path>/opentelemetry-javaagent.jar \
-Dotel.exporter.otlp.endpoint=https://ingest.{region}.signoz.cloud:443 \
-Dotel.exporter.otlp.headers='signoz-access-token=<SIGNOZ_INGESTION_KEY>' \
-Dotel.resource.attributes='service.name=<app_name>'"
```
The " \ " will just be used to give a better environment variable visibility in the `standalone.conf` file. Right ?

Reply:
The latter adds `\` inside the quoted string (unlike other commands), which I think doesn't work. We should try to run and verify these instructions work.

Reply:
Okay. Got it. That probably won't work.
I'll see how to run and verify this.

Reply:
Yes, please verify. If you need any help, post in engineering-pod. since these are the first steps customers follow in onboarding, we should ensure they work.